### DISCUSSION TITLE
如果doris既作为离线数仓，又作为实时数仓，还需要hdfs吗

### DISCUSSION THREAD

**Discussion by yaotengyuan** [apache/doris#23110] [RFC]
如果doris既作为离线数仓，又作为实时数仓，还需要hdfs吗，第一次做数仓搭建，希望大佬们多多指导

**Comment by GaoYang20** [comment]
不需要，数据量不是特别大的话，完全撑得住，我这边目前使用的是 doris实现的 离线+实时数仓，1个fe，3个be，16+64的配置。
离线：datax + doris + dolphinscheduler
实时：doris + flink（flinkcdc同步，flink sql/DataStream做实时计算或者宽表）

**Reply by hli191** [reply]
你这个16+64G的配置承载了多少的数据量？宽表计算的时候需要反查Doris吗？宽表实时计算实效性是怎样的？请教一下。
